hooks/useTBAPurchase.tsx

```diff
@@ -37,11 +39,21 @@ const useTBAPurchase = () => {
         zoraTotalPrice.toString(),
       ) as any
       const tba = getAccount(zoraNextTokenId)
-      const soundMintCall = await getSoundMintCall(tba)
+      const tbaInitializationCall = await getTBAInitializeCall(tba)
```

Comment:
I don't think we need to `await` this 
```suggestion
      const tbaInitializationCall = getTBAInitializeCall(tba)
```

Reply:
fixed ✅